src/tls/Makefile

```diff
@@ -79,7 +79,7 @@ FSTAR_HINTS ?= --use_hints --use_hint_hashes --record_hints
 #      verify... see https://github.com/FStarLang/FStar/issues/1652
 FSTAR_NO_FLAGS = $(FSTAR_HOME)/bin/fstar.exe $(FSTAR_HINTS) \
   --odir $(OUTPUT_DIR) --cache_checked_modules $(FSTAR_INCLUDES) --cmi \
-  --already_cached 'Prims FStar LowStar C Spec.Loops TestLib WasmSupport Hacl EverCrypt Vale LowParse Lib Spec -FStar.Old.Endianness -FStar.Test -C.Compat' \
+  --already_cached 'Prims FStar LowStar C Spec.Loops TestLib WasmSupport LowParse -FStar.Old.Endianness -FStar.Test -C.Compat' \
```

Comment:
won't you have "did not expect module XXX to be checked" errors with this?

Reply:
They are only warnings e.g. `Did not expected EverCrypt.HKDF to be already checked, but found it in an unexpected location ...` (sic). I believe it will behave as before except for these warnings.